ui/packages/shared/components/src/DateTimeRangePicker/DateTimeRangePickerText/index.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 The Parca Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import cx from 'classnames';
+
+import RelativeDatePicker from '../RelativeDatePicker';
+import {DateTimeRange, DateUnion, formatDateStringForUI} from '../utils';
+
+interface DateTimeRangePickerTextProps {
+  range: DateTimeRange;
+  onClick: () => void;
+  isActive: boolean;
+  onRangeSelection: (range: DateTimeRange) => void;
+}
+
+const DateTimeRangePickerText = ({
+  range,
+  onClick,
+  isActive,
+  onRangeSelection,
+}: DateTimeRangePickerTextProps): JSX.Element => {
+  const isRelativeRange = range.from.isRelative();
+  const dateString = `${formatDateStringForUI(range.from)} → ${formatDateStringForUI(range.to)}`;
```

Comment:
Maybe we can try something like https://github.com/parca-dev/parca/blob/c56e0c29debe2bfaf9373802db4f136aba1f7872/ui/packages/shared/components/src/DateTimeRangePicker/utils.ts#L115-L125 here so that the text is a bit concise.

Reply:
yeah this is better, thanks for the suggestion